node/src/sub_lib/accountant.rs

```diff
@@ -292,4 +395,39 @@ mod tests {
             }
         );
     }
+
+    #[test]
+    fn messages_with_services_provided_are_correctly_implemented() {
+        let service = ServiceProvided {
+            timestamp: SystemTime::now(),
+            paying_wallet: make_wallet("abc"),
+            payload_size: 0,
+            service_rate: 0,
+            byte_rate: 0,
+        };
+        let exit_service_msg = ReportExitServiceProvidedMessage {
+            service: service.clone(),
+        };
+        let routing_service_msg = ReportRoutingServiceProvidedMessage { service };
```

Comment:
This test will change slightly. You should consider a test with 2 cases along with a helper function.